Modules/Sources/WordPressUI/Views/Users/ViewModel/DisplayUser.swift

```diff
@@ -42,6 +42,14 @@ public struct DisplayUser: Identifiable, Codable {
         self.biography = biography
     }
 
+    public static func == (lhs: DisplayUser, rhs: DisplayUser) -> Bool {
+        lhs.id == rhs.id
+    }
+
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
```

Comment:
This isn't an ideal implementation of  `Equatable` and `Hashable` for this struct. I'd expect it to perform a per-field check. The id comparison is what `Identifiable` is for, which it already supports.

Reply:
I added `Hashable` here so that it can be set as the selection of `Picker` view. In that context, `id` is sufficient for hash code implementation. But I have removed the custom implementation in https://github.com/wordpress-mobile/WordPress-iOS/pull/23791/commits/260c8a7329e1199a05e6b10b421798908cbf7cf2.